README.md

```diff
@@ -119,7 +119,32 @@ Sometimes it may happen that you need to process the relationship before renderi
 ```ruby
 class BlogPostSerializer < BaseSerializer
   lazy_has_many :comments do |serializer|
-    serializer.lazy_comments.map(&:decorate)
+    -> { serializer.lazy_comments.map(&:decorate) }
+   end
+end
+```
+
+Keep attention that custom finder should be returned in a form of lambda,
```

Comment:
```suggestion
If you're using `include_data` AMS setting make sure that your association blocks return a lambda in order to avoid redundant SQL queries when `include_data` AMS setting evaluates to `false`:
```

Reply:
Again, typically `include_data` will be turned off a way further from the project beginning, on the very last phase of `make it work, make it right, make it fast` ;) I think it's good to make an accent how to do the things better right from beginning )

Reply:
Ok